SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -1,24 +1,31 @@
 #!/bin/bash
 
+# Create a middle layer for installing Spark extensions, putting them apart from the user environment
+uv venv $SWAN_ENV --seed 2>&1
```

Comment:
Why --seed?

Reply:
uv, by default, doesn't install setuptools, wheels and mainly, pip. Which means when the users does `pip list`, the pip binary is only found under the `/opt/conda/...`. This wasn't noticeable in accpy environments because the work of reacreating the pip was being already done by accpy. In venv that doesn't happen because we're creating the venv with uv. What `--seed` does is installing these 3 packages, which leads us to actually get a dry environment when doing `pip list`.

Reply:
If you create the env with venv. venv installs pip?

Reply:
Yes, creating it with `python -m venv` brings the pip with it.

Reply:
```
Creating virtual environment with seed packages at: test1
 + pip==24.3.1
 + setuptools==75.6.0
 + wheel==0.45.1
Activate with: source test1/bin/activate

real    0m0.029s
user    0m0.011s
sys     0m0.027s
```
```
bash-5.1$ time python -m venv test2

real    0m2.748s
user    0m2.426s
sys     0m0.270s
```

Reply:
Is the uv time cold or hot? It still needs to download those three packages, and it took 29 ms?

Reply:
Considering the session I was in was creating using pip install because I used a requirements.in file, uv may have used pip cache to download them. Fine, i can explicitly say i dont want to use cache.

```
bash-5.1$ time uv venv test1 --seed --no-cache
Using CPython 3.11.9 interpreter at: /opt/conda/bin/python
Creating virtual environment with seed packages at: test1
 + pip==24.3.1
 + setuptools==75.6.0
 + wheel==0.45.1
Activate with: source test1/bin/activate

real    0m0.295s
user    0m0.101s
sys     0m0.120s
```
```
bash-5.1$ time python -m venv test2

real    0m2.772s
user    0m2.454s
sys     0m0.259s
```